app/mailers/user_mailer.rb

```diff
@@ -272,6 +272,19 @@ def in_person_failed(user, email_address, enrollment:)
     end
   end
 
+  def in_person_failed_fraud(user, email_address, enrollment:)
+    with_user_locale(user) do
+      @presenter = Idv::InPerson::VerificationResultsEmailPresenter.new(
+        enrollment: enrollment,
+        url_options: url_options,
+      )
+      mail(
+        to: email_address.email,
+        subject: t('user_mailer.in_person_failed_suspected_fraud.subject', app_name: APP_NAME),
```

Comment:
```suggestion
        subject: t('user_mailer.in_person_failed_suspected_fraud.subject'),
```
There's no app_name variable in this translation.